rever/main.xsh

```diff
@@ -10,6 +11,30 @@ from rever import environ
 from rever.dag import find_path
 
 
+NEWS_TEMPLATE = '''
```

Comment:
This doesn't need to still be here does it?

Reply:
Yes sorry